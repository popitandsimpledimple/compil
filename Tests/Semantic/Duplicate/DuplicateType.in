type int = Integer;
type int = Double;
begin end.