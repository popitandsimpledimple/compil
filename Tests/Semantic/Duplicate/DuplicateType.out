(2,6) Duplicate identifier 'int'